railto/TwigAssetVersionExtension
# Add support for Twig 3 and PHP 8, update tests to phpunit 9

- Add support for Twig 3 (tested with Slim 4)
- Add support for PHP 8, require PHP 7.3+ for compatibility with Slim 3 and phpunit 9
- Add shortcuts for code style check and testing in `composer.json`
- Update tests to latest phpunit version
